pkg/controller.v1/common/job_controller.go

```diff
@@ -177,13 +178,27 @@ var GenSchedulerPluginsSetupFunc = func(c client.Client) GangSchedulingSetupFunc
 	}
 }
 
+var GenKoordinatorSetupFunc = func(c client.Client) GangSchedulingSetupFunc {
+	return func(jc *JobController) {
+		jc.Config.GangScheduling = GangSchedulerKoordinator
+		jc.PodGroupControl = control.NewKoordinatorControl(c)
+	}
+}
+
 var GenNonGangSchedulerSetupFunc = func() GangSchedulingSetupFunc {
 	return func(jc *JobController) {
 		jc.Config.GangScheduling = GangSchedulerNone
 		jc.PodGroupControl = nil
 	}
 }
 
+var GenKoordinatorSetupFunc = func(c client.Client) GangSchedulingSetupFunc {
+	return func(jc *JobController) {
+		jc.Config.GangScheduling = GangSchedulerKoordinator
+		jc.PodGroupControl = control.NewKoordinatorControl(c)
+	}
+}
+
```

Comment:
Why are there the same functions (L181-L187)?

Reply:
Done